src/scripts/public/notification-banner.js

```diff
@@ -0,0 +1,103 @@
+import { createElement, qs } from './utils';
+
+/**
+ * General Block Theme window event management and DOM manipulation.
+ */
+const domReady = () => {
+	/*
+	 * SafarIE bug requires 0ms timeout.
+	 */
+	setTimeout(function() {
+		const body = qs('body');
+		let notificationsEnabled = window.site.noticeEnabled;
+		const notificationsHomepageOnly = window.site.noticeHomepageOnly;
+		const notificationsLabelBold = window.site.noticeLabelBold;
+		const notificationsLabelBig = window.site.noticeLabelBig;
+		const notificationsStatus = window.site.noticeStatus;
+		const notificationsContent = window.site.noticeContent;
+		const notificationsButtonLabel = window.site.noticeButtonLabel;
+		const notificationsButtonEnabled = window.site.noticeButtonEnabled;
+		const notificationsButtonAriaLabel = window.site.noticeButtonAriaLabel;
+		const notificationsButtonLink = window.site.noticeButtonLink;
+		const notificationsButtonUtmCampaign =
+			window.site.noticeButtonUtmCampaign;
+
+		if (
+			notificationsEnabled &&
+			notificationsHomepageOnly &&
+			body.classList.contains('home')
+		) {
+			notificationsEnabled = true;
```

Comment:
Could remove the above check, then the conditional on lines 32-38 and do this?

```suggestion
			notificationsEnabled = body.classList.contains('home') ? true: false;
```